openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -1090,17 +1090,6 @@ def _apply_third_party_auth_overrides(self, request, form_desc):
                                     instructions="",
                                 )
 
-                    # Hide the confirm_email field
```

Comment:
Are we hiding the field some other way now?

Reply:
This code was added by me but it did not work so I removed it.  and the field is now being removed [here](https://github.com/edx/edx-platform/pull/24314/files#diff-07e31524ab9054c0f644873f7001dc15R174)